components/plurals/src/operands.rs

```diff
@@ -33,45 +33,46 @@ use fixed_decimal::FixedDecimal;
 ///
 /// ```
 /// use icu::plurals::PluralOperands;
-/// assert_eq!(PluralOperands {
-///    i: 2,
-///    v: 0,
-///    w: 0,
-///    f: 0,
-///    t: 0,
-///    c: 0,
-/// }, PluralOperands::from(2_usize))
+/// assert_eq!(PluralOperands::new(
+///    2, // i
+///    0, // v
+///    0, // w
+///    0, // f
+///    0, // t
+///    0, // c
+/// ), PluralOperands::from(2_usize))
```

Comment:
`question`: why are you changing this? I feel like this is a good example where ordered based list is inferior to key=>value map of fields. Since it is stable and exhaustive, can we keep the old API?

Reply:
@sffc left a comment saying it wasn't stable. I would also prefer to not use a list based api here.

Reply:
I think it should be stable.